packages/gatsby-plugin-sitemap/src/gatsby-ssr.js

```diff
@@ -3,17 +3,20 @@ import { withPrefix } from "gatsby-link"
 import { defaultOptions } from "./internals"
 
 exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
-  let { output } = { ...defaultOptions, ...pluginOptions }
-  if (output.charAt(0) !== `/`) {
-    output = `/` + output
-  }
+  let { output, addLinkToHead } = { ...defaultOptions, ...pluginOptions }
+
+  if (addLinkToHead) {
```

Comment:
Maybe this could just return early instead of having nested if statements?

```
if (addLinkToHead === false) return

// ... carry on 
```

Not a big deal though.